agent/engine/dockerclient/versionsupport_windows.go

```diff
@@ -11,15 +12,21 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package functional_tests
+package dockerclient
 
-import (
-	"testing"
+func getAgentVersions() []DockerVersion {
+	return []DockerVersion{
+		Version_1_17,
+		Version_1_18,
+		Version_1_19,
+		Version_1_20,
+		Version_1_21,
+		Version_1_22,
+		Version_1_23,
+		Version_1_24,
```

Comment:
Why is 1.24 included on Windows but not Linux?  Should this be 1.23 on both instead?